src/modules/kvs-watch/kvs-watch.c

```diff
@@ -1115,6 +1117,20 @@ static void watcher_cancel_all (struct watch_ctx *ctx,
     char *name;
     struct ns_monitor *nsm;
 
+    /* if canceling can do lookup to avoid iterating over all
+     * namespaces */
```

Comment:
wording in comment confused me. Should it be "if canceling, do lookup to avoid iterating..."? (i.e. switch "can" for a comma?)